src/blocking/dac.rs

```diff
@@ -0,0 +1,14 @@
+//! Digital-analog conversion traits
+
+
+//! A trait used to identify a digital-to-analog converter, and its
+//! most fundamental features.
+
+pub trait SingleChannelDac {
+    /// Error type returned by DAC methods
+    type Error;
+    type Word;
+
+    /// Output a constant signal, given a bit word.
+    fn try_set_value(&mut self, value: Self::Word) -> Result<(), Self::Error>;
+}
```

Comment:
```suggestion
//! Trait representing a single-channel digital-to-analog converter (DAC).
pub trait SingleChannelDac<Word> {
    /// Error type returned by DAC methods
    type Error;

    /// Output a constant signal, given a bit word.
    fn try_set_value(&mut self, value: Word) -> Result<(), Self::Error>;
}
```
Putting the word as a type parameter we can have different implementations for different types.